apps/remix-ide/src/app/files/fileManager.ts

```diff
@@ -376,8 +376,8 @@ class FileManager extends Plugin {
         const provider = this.fileProviderOf(path)
 
         provider.resolveDirectory(path, (error, filesProvider) => {
-          if (error) reject(error)
-          resolve(filesProvider)
+          if (error) return reject(error)
```

Comment:
why return that?

Reply:
without `return reject`, it goes on to resolve(null) as filesProvider.